client/src/components/SectionNav/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { connect } from 'react-redux'
+
+import uiSetSection from 'actions/uiSetSection'
+import Button from 'components/Button'
+
+const SectionNav = ({ sections, ui, uiSetSection }) =>
+  <div>
```

Comment:
@albatrocity might use a Box here with some sweet childSpacing to give these little puppies some personal space.